tox.ini

```diff
@@ -18,7 +18,10 @@ setenv = VIRTUAL_ENV={envdir}
 deps =  -r{toxinidir}/requirements.txt
         -r{toxinidir}/test-requirements.txt
 commands =
-  python -m testtools.run discover tests
+  python -m testtools.run tests.test_api_runner
```

Comment:
Could you please add a comment here as to why you had to split them up. That way, in the future someone won't try to combine this back to just "discover tests"